### PR TITLE
docs(KFLUXINFRA-930): README: Add troubleshooting

### PR DESCRIPTION
**Pull request by ifireball**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -169,3 +169,13 @@ likely to be resolved in the future.
   referring to may be changed. In both those cases, resources owned by the
   *ProjectDevelopmentStream* but not defined by the new template do not get
   deleted.
+
+## Troubleshooting 
+
+* When a *ProjectDevelopmentStream* that refers to a
+  *ProjectDevelopmentStreamTemplate* is created or modified, and the resources
+  defined by the template fail to be created. The issues with the template
```
Comment (yftacherzog) [inline README.md:177]:
```suggestion
  defined by the template fails to be created, the issues with the template
```

File: README.md [hunk 2]
```diff
@@ -169,3 +169,13 @@ likely to be resolved in the future.
...
+  application process may be seen by inspecting the Events associated with the
+  *ProjectDevelopmentStream* object. The events can be seen by running the
+   `kubectl describe` or `oc describe` commands on the
+  *ProjectDevelopmentStream* resource
```
Comment (yftacherzog) [inline README.md:181]:
```suggestion
  *ProjectDevelopmentStream* resource.
```

**Review by yftacherzog — COMMENTED** [review]
Minor comments

File: README.md (hunk 1 above)
Comment (yftacherzog) [reply README.md:177]: (I edited this comment)

Comment (ifireball) [reply README.md]: applied.

File: README.md (hunk 2 above)
Comment (ifireball) [reply README.md]: applied.

**Review by yftacherzog — APPROVED** [review]
lgtm

**Pull request merged**